doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -137,10 +108,13 @@ Adding or editing tutorials as Jupyter notebooks
 ------------------------------------------------
 
 Under the ``doc/source/`` folder of the SciPy tree you can find a few
-documents written in MyST-NB_ format. These files are executable, meaning that
-their content is executed when the SciPy documentation is built (locally or on
-CI) and any outputs generated by the execution are rendered in the final HTML
-files.
+documents written in MyST-NB_ markdown format. These files are executable,
```

Comment:
Nitpicking: the name of the format is MyST Markdown, and MyST-NB is the Sphinx extension that allows adding documents bearing this format to websites built with Sphinx. Isn't it?

Reply:
Fair points! I reworded this.